### DISCUSSION TITLE
Alternatives to Stylus?

### DISCUSSION THREAD

**Discussion by cheater** [openstyles/stylus#1563] [Q&A] [answered]
Hi, I like Stylus a bunch, but I was wondering what alternatives there are to it? I'm trying to make an informed decision about what I could be using - no shade thrown at the project. I know there's something called "Stylish" but I heard it's abandoned / recommended against and I wasn't able to find an open source repository for it. The browser I use is Firefox if that makes a difference.

Thanks!

**Comment by tophf** [comment, marked as answer]
See https://github.com/openstyles/stylus/wiki/Stylish-alternatives

Stylish is now owned by SimilarWeb, a company that collects and sells user's behavioral data, and that's the main purpose of Stylish, while showing styles is the compensation.

**Comment by cheater** [comment]
Thank you!
